support-e2e/tests/monthlyContribution.test.ts

```diff
@@ -4,58 +4,58 @@ import { email, firstName, lastName } from "./utils/users";
 
 test.beforeEach(async ({ page, context, baseURL }) => {
   const baseUrlWithFallback = baseURL ?? "https://support.theguardian.com";
-  const pageUrl = `${baseUrlWithFallback}/uk/contribute`;
+  // We should remove the forcing into the ab test once this has been made live
+  const pageUrl = `${baseUrlWithFallback}/uk/contribute#ab-twoStepCheckoutWithNudgeBelow=variant_a`;
 
   const domain = new URL(pageUrl).hostname;
   await setTestCookies(context, firstName, domain);
-  await page.goto(pageUrl, {waitUntil: "networkidle"});
+  await page.goto(pageUrl, { waitUntil: "networkidle" });
 });
 
 test.describe("Sign up for a Recurring Contribution (New Contributions Flow)", () => {
   test("Monthly contribution sign-up with Stripe - GBP", async ({ page }) => {
-    const monthlyTab = "#MONTHLY";
-    await page.locator(monthlyTab).click();
+    await page.getByRole("tab").getByText("Monthly").click();
+    await page.getByRole("button", { name: "Continue to checkout" }).click();
 
-    await page.locator("#email").type(email);
-    await page.locator("#firstName").type(firstName);
-    await page.locator("#lastName").type(lastName);
+    await page.getByLabel("Email address").type(email);
+    await page.getByLabel("First name").type(firstName);
+    await page.getByLabel("Last name").type(lastName);
 
-    await page.locator("#qa-credit-card").click();
+    await page.getByLabel("Credit/Debit card").click();
```

Comment:
And I think , we can use  await page.getByRole('radio', { name: 'Credit/Debit card' }).click(); here as it's a radio button, but this is also fine